llms/mistral/mistralmodel.go

```diff
@@ -0,0 +1,236 @@
+package mistral
+
+import (
+	"context"
+	"errors"
+	"os"
+
+	sdk "github.com/gage-technologies/mistral-go"
+	"github.com/tmc/langchaingo/callbacks"
+	"github.com/tmc/langchaingo/llms"
+	"github.com/tmc/langchaingo/schema"
+)
+
+// LLM is a mistral LLM implementation.
+type LLM struct {
+	CallbacksHandler callbacks.Handler
```

Comment:
not necessary but given how many bytes/tokens will flow through here I feel like some comments would be fitting.